chromium_src/chrome/browser/policy/configuration_policy_handler_list_factory.cc

```diff
@@ -25,6 +26,10 @@
 #include "brave/components/ipfs/pref_names.h"
 #endif
 
+#if BUILDFLAG(ENABLE_BRAVE_VPN)
+#include "brave/components/brave_vpn/common/pref_names.h"
+#endif
+
 namespace {
 
 const policy::PolicyToPreferenceMapEntry kBraveSimplePolicyMap[] = {
```

Comment:
nit: we can move this into something like `//brave/browser/policy/brave_simple_policy_map.h/.cc`.

Reply:
https://github.com/brave/brave-browser/issues/29468